pkg/render/node.go

```diff
@@ -978,6 +980,10 @@ func (c *nodeComponent) bpffsInitContainer() corev1.Container {
 			// so that it outlives the init container.
 			MountPropagation: &bidirectional,
 		},
+		{
+			MountPath: "/node-proc",
```

Comment:
I'm guessing the mount command we do with nsenter needs write access to the volume. Even though I think the answer is no, could this VolumeMount be done ReadOnly?

Reply:
I'll try it.

Reply:
Setting it to `ReadOnly` works fine. Done!